pkg/cmd/extension/manager.go

```diff
@@ -591,8 +597,155 @@ func (m *Manager) Create(name string) error {
 	}
 	dir := filepath.Join(wd, name)
 	addCmd := m.newCommand(exe, "-C", dir, "--git-dir="+filepath.Join(dir, ".git"), "add", name, "--chmod=+x")
+	return addCmd.Run()
+}
+
+func (m *Manager) otherBinScaffolding(gitExe, name string) error {
+	err := os.MkdirAll(filepath.Join(name, ".github", "workflows"), 0755)
+	if err != nil {
+		return err
+	}
+	workflow := heredoc.Doc(`
+	name: release
+	on:
+	  push:
+	    tags:
+	      - "v*"
+	permissions:
+	  contents: write
+	
+	jobs:
+	  release:
+	    runs-on: ubuntu-latest
+	    steps:
+	      - uses: actions/checkout@v2
+	      - uses: cli/gh-extension-precompile@v1
+		  	with:
+			  build_script_override: "script/build.sh"
+	`)
+	workflowPath := filepath.Join(".github", "workflows", "release.yml")
```

Comment:
In the interest of readability and easier maintenance, could we have these files live as physical files in our repository and bake them into the gh binary at compile time using [golang's embed functionality](https://pkg.go.dev/embed@go1.16)? Then we could recursively iterate through contents of a whole directory and create files and their paths as necessary.